docs/source/federation-versions.mdx

```diff
@@ -0,0 +1,565 @@
+---
+title: Federation version changelog
+description: Understand changes between federation versions
+---
+
+## v2.3
+
+**Directive changes**
+
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr>
+<td>
+
+##### `@interfaceObject`
+
+</td>
+<td>
+
+```graphql
+directive @interfaceObject on OBJECT
+```
+
+</td>
+</tr>
+
+</tbody>
+</table>
+
+**Subgraph changes**
+
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr>
+<td>
+
+Interfaces
+
+</td>
+<td>
+
+`@key` directive can now be used on interface definitions.
```

Comment:
I would instead make this a second _directive_ change in the table _above_ for simplicity, where the left column value is `@key` and the description is "Composition now allows `@key` to be applied to interface definitions."